src/script/script_instance.hpp

```diff
@@ -261,11 +261,11 @@ class ScriptInstance {
 
 	/**
 	 * Load squirrel scripts to emulate an older API.
-	 * @param api_version: API version to load scripts for
-	 * @param dir Subdirectory to find the scripts in
-	 * @return true iff script loading should proceed
+	 * @param dir Subdirectory to find the scripts in.
+	 * @param api_versions List of available versions of the script type.
+	 * @return true iff script loading should proceed.
 	 */
-	bool LoadCompatibilityScripts(const std::string &api_version, Subdirectory dir);
+	bool LoadCompatibilityScripts(Subdirectory dir, const std::initializer_list<std::string_view> &api_versions);
```

Comment:
I wonder if it is better to pass a std::span rather than a std::initialiser_list reference.

Reply:
Rb suggested the same thing, but wasn't sure it worked. I couldn't get it to work. Found https://quuxplusone.github.io/blog/2021/10/03/p2447-span-from-initializer-list/. Suggests it can't work.

But I would love to have an `std::span` here instead :D So suggestions what I am doing wrong are welcome!

Reply:
Turns out: `std::span<const ...` was the trick!